float64[7] JointPos
float64[7] JointVel
float64[7] JointAcc
float64[7] Torque
float64[16] EndPos
float64[6] EndAcc
float64[7] MotorPos

